guide/07-working-with-bigdata/10-extending_geoanalytics_with_custom_tools.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>As the GeoanAlytics and pyspark packages both work with Spark DataFrames, you can combine tools from each into a single analysis workflow.</p><p> </p><p>In the example below, a <a href="https://spark.apache.org/docs/2.2.0/ml-classification-regression.html#naive-bayes" target="_blank">Naive Bayes classifier</a> included with pyspark is trained to predict plant species using leaf and petal measurements. The classifier is applied to a dataset of unlabeled plant sightings to predict the species of each specimen. Afterward, the <a href="https://developers.arcgis.com/rest/services-reference/enterprise/create-buffers.htm#GUID-8563906D-2398-4969-A0CF-7A3712C96BE5" target="_blank">Create Buffers</a> GeoAnalytics tool is used to create a map of habitat area for each plant species. The tool result is then written to ArcGIS Enterprise.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>